setup/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "setup-notation",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
```

Comment:
It is better to upgrade to TypeScirpt. We can do that in another PR.

Reply:
Should I open an issue for this on the repo?

Reply:
We can create an issue in this repo and improve it in a follow-up PR

Reply:
Issue created: https://github.com/notaryproject/notation-action/issues/12